src/Main.elm

```diff
@@ -48,12 +51,16 @@ subscriptions model =
 
 viewDocument : Model -> Browser.Document Msg
 viewDocument model =
-    { title = "[cCc] App title", body = [toUnstyled (view model) ] }
+    { title = (translate model.language SiteTitle), body = [toUnstyled (view model) ] }
```

Comment:
We'll probably want to alias this to t where ever used to shorted the code. But not an issue yet.